gnmi_server/server.go

```diff
@@ -230,6 +233,9 @@ func (s *Server) Subscribe(stream gnmipb.GNMI_SubscribeServer) error {
 		return nil, status.Error(codes.PermissionDenied, msg)
 	}
 	*/
+	if !s.config.TelemetryEnable {
```

Comment:
>TelemetryEnable [](http://example.com/codeflow?start=14&length=15)

I am not clear about the necessity for disabling subscribe. Is there a use case of subscribing a path in ConfigDB? if yes, no need to add option.